examples/childchain-inflight-exit.js

```diff
@@ -158,22 +202,38 @@ async function inflightExitChildChain () {
   console.log(`Final alice rootchain balance: ${aliceRootchainBalance}`)
   console.log(`Final bob rootchain balance: ${bobRootchainBalance}`)
 
-  console.log(`Final alice childchain balance: ${aliceChildchainBalanceArray.length === 0 ? 0 : aliceChildchainBalanceArray[0].amount}`)
-  console.log(`Final bob childchain balance: ${bobChildchainBalanceArray.length === 0 ? 0 : bobChildchainBalanceArray[0].amount}`)
+  console.log(
+    `Final alice childchain balance: ${
+      aliceChildchainBalanceArray.length === 0
+        ? 0
+        : aliceChildchainBalanceArray[0].amount
+    }`
+  )
+  console.log(
+    `Final bob childchain balance: ${
+      bobChildchainBalanceArray.length === 0
+        ? 0
+        : bobChildchainBalanceArray[0].amount
+    }`
+  )
 
   bobUtxos = await childChain.getUtxos(bobAddress)
   bobUtxoToExit = bobUtxos[0]
 
   aliceUtxos = await childChain.getUtxos(aliceAddress)
   aliceUtxoToExit = aliceUtxos[0]
 
-  console.log('Alice still has a UTXO as she has not exited yet. But Bob has no remaining UTXOs as he already exited his UTXO.')
+  console.log(
+    'Alice still has a UTXO as she has not exited yet. But Bob has no remaining UTXOs as he already exited his UTXO.'
+  )
 
-  console.log(`Alice's UTXOS = ${JSON.stringify(aliceUtxoToExit, undefined, 2)}`)
+  console.log(
+    `Alice's UTXOS = ${JSON.stringify(aliceUtxoToExit, undefined, 2)}`
+  )
   console.log(`Bob's UTXOs = ${JSON.stringify(bobUtxoToExit, undefined, 2)}`)
 }
 
-(async () => {
+;(async () => {
```

Comment:
intentional right? compiler wil run previous block without it?

Reply:
can be removed, probably added from linting.